### ISSUE TITLE
preg_match with modifier 'e' is deprecated.

### ISSUE THREAD

**Issue by hrniels** [script-solution/Boardsolution#75]
Instead, one should use preg_match_callback, which is much cleaner anyway because you don't write the code as a string which is than eval'd but specify a function to call.

I would have fixed that right away, but the problem is that it gets really annoying to write a function for each of those cases (look in fws/template/parser.php) and unfortunately, anonymous functions are only supported in PHP >= 5.3. This would raise the requirements from currently 5.2 to 5.3 which we definitely shouldn't do in bugfix releases.

So, I guess we should stick with the current way of just disabling deprecated-messages and fix it with anonymous functions in BS v1.50, which will then require PHP 5.3 (or later, we'll see).

**Comment by Rafioso** [comment]
@hrniels 
preg_match with modifier 'e' is **NOT** deprecated. Maybe you meant preg_replace. And this is already fixed in c7352e2a219c7764dce9202d59574c0aa2f8e850

**Comment by hrniels** [comment]
Yes, I meant `preg_replace`. The 'e' modifier makes no sense for `preg_match` :)
I'm not sure if there are other places left, but it might indeed already be fixed.
